frontend/pages/ManageControlsPage/SetupExperience/_styles.scss

```diff
@@ -16,4 +16,8 @@
     padding: 64px 170px;
     margin: 0;
   }
+
+  .side-nav__nav-list {
+    top: 0;
+  }
```

Comment:
LGTM and is consistent with other side navs, and with the Figma, but would be good to get the +1 from @rachaelshaw or @noahtalerman since there's no dev note for it and it will affect the Scripts tab as well.

Before:

<img width="1108" height="371" alt="image" src="https://github.com/user-attachments/assets/fd934f3f-8e61-4500-bdd0-0a22aa22b29f" />

After:

<img width="1126" height="405" alt="image" src="https://github.com/user-attachments/assets/48e23a03-7226-4302-8b60-8d19b8fe1dcb" />